data/json/items/gun/shot.json

```
@@ -147,8 +147,8 @@
     "copy-from": "shotgun_pump",
     "looks_like": "remington_870",
     "type": "GUN",
-    "name": { "str": "24-round tube shotgun" },
-    "description": "This shotgun uses a system of 2 tube magazines to carry 24 rounds of 12 gauge shot.  Each tube has to be loaded separately, but this offers the option of loading different ammunition for different situations.",
+    "name": { "str": "high-capacity Kel-Tec pump-action shotgun" },
```

ditto on brand names.
```suggestion
    "name": { "str": "high-capacity pump-action shotgun" },
```